app/classes/generate_new_week_assignments.rb

```diff
@@ -1,5 +1,5 @@
-module AssignmentGenerationHelper
-  def generate_new_week_assignments(week)
+class GenerateNewWeekAssignments
+  def new(week)
```

Comment:
The `new` method is like a keyword method and should never be defined in a Ruby class. We use `initialize` instead which is called by `new`. The `initialize` method should only set up the object and we should have a `call` method that does the real work of finding employees and creating assignments.

Finally you should extract some of the code here into private methods in accordance with the single responsibility principle (I think this is in the first or second chapter of that Ruby book 😄). The named private methods will help me understand what your code is doing (it's hard to follow without digging into the logic).